src/OpenLoco/Windows/CompanyWindow.cpp

```diff
@@ -1612,10 +1613,11 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
             commonWidgets(636, 319, StringIds::title_company_finances),
             makeWidget({ 133, 45 }, { 499, 215 }, WidgetType::scrollview, WindowColour::secondary, Scrollbars::horizontal),
             makeStepperWidgets({ 87, 264 }, { 100, 12 }, WidgetType::wt_17, WindowColour::secondary, StringIds::company_current_loan_value),
+            makeWidget({ 320, 264 }, { 204, 12 }, WidgetType::checkbox, WindowColour::secondary, StringIds::loan_autopay, StringIds::tooltip_select_main_colour), // loan_autopay
```

Comment:
Rather than using `StringIds::tooltip_select_main_colour`, could you add a dedicated tooltip for this new widget? It can just explain in a sentence or two that funds will be deducted automatically at the start of the month, or some such.

Reply:
good pickup, this was a copy/paste mistake, fixed